zest/releaser/vcs.py

```diff
@@ -318,7 +320,7 @@ def list_files(self):
         works is handy for the vcs.txt tests.
         """
         files = []
-        for dirpath, dirnames, filenames in os.walk('.'):
+        for dirpath, dirnames, filenames in os.walk(utils.BASE):
```

Comment:
I think this one should not be done. With this change, zest.releaser will look for a changelog file in utils.BASE, so it will miss the changelog file in the main directory.